docs/help.md

```diff
@@ -220,6 +220,40 @@ correctly with the use of [`chcp`](https://ss64.com/nt/chcp.html) by
 executing in the terminal `chcp 65001`. This is a known limitation of
 Node.js.
 
+It is also possible to change the console encoding from within nodejs using the [ffi-napi](https://www.npmjs.com/package/ffi-napi) module (or any other ffi module you like, [koffi](https://www.npmjs.com/package/koffi) for example).
+
+### Changing encodings from within Nodejs
+
+```js
+log.info("Hello World!"); //INFO (11856): Hello World!
+log.info("Привет мир!"); //INFO (11856): ╨Я╤А╨╕╨▓╨╡╤В ╨╝╨╕╤А!
+log.info("你好世界！"); //INFO (11856): ф╜ахе╜ф╕ЦчХМя╝Б
+log.info("Halo Wɔl!"); //INFO (11856): Halo W╔Фl!
+```
+Use the ffi module to call the native functions [SetConsoleOutputCP](https://learn.microsoft.com/en-us/windows/console/setconsoleoutputcp) and [SetConsoleCP](https://learn.microsoft.com/en-us/windows/console/setconsolecp).
```

Comment:
This sentence seems out of place. There should be something leading in to it to describe what using this APIs will accomplish.